### ISSUE TITLE
RGB_FILLED failed but depth successed

### ISSUE THREAD

**Issue by rainprob** [StanfordVL/GibsonEnv#129]
![fail](https://user-images.githubusercontent.com/63865505/125779071-ebf221d5-cd31-44df-8ad6-1528d5222bdd.PNG)

**Comment by rainprob** [comment]
I tried RGB_PREFILLED, NORMAL, and SEMENTICS. 
RGB_PREFILLED can display the correct content but it still has some wrong pixels, the results of NORMAL and SEMENTICS are both black. _
![fail1](https://user-images.githubusercontent.com/63865505/125787622-f50796f4-7620-4919-9052-d1d145808506.PNG)

**Comment by rainprob** [comment]
I tried all solutions in issues, such as checking the version of torchvision and torch, kill depth_render process, and others. But these all don't work.

**Comment by Berk035** [comment]
Hello @rainprob ,

I recommend to reinstall the GibsonEnv clearly. You can try to install docker version instead of building source. Have a nice day.

**Comment by fxia22** [comment]
@rainprob I observe similar issues when the `gibson/core/render/render_cuda_f.so` is not correctly compiled with nvcc. Can you check your `nvcc --version` is the same as `nvidia-smi` version?

**Comment by rainprob** [comment]
@fxia22 
You are right. My nvcc --version is 11.0, however nvidia-smi version is 11.2. So, how can I solve that? Make them both versions same？

**Comment by rainprob** [comment]
@Berk035 
Thanks for your reply.

**Comment by rainprob** [comment]
> @fxia22
> You are right. My nvcc --version is 11.0, however nvidia-smi version is 11.2. So, how can I solve that? Make them both versions same？

In gibson docker container, nvcc --version is 9.0 and nvdia-smi version is 11.2.

**Comment by fxia22** [comment]
It seems that you have multiple versions of cuda installed in `/usr/local`, you can expose the version used in `nvidia-smi` by soft linking `cuda` to it. For example:
![image](https://user-images.githubusercontent.com/5158896/127429832-cc9ff20e-9e13-4e3e-95f0-f505436c94b6.png)

![image](https://user-images.githubusercontent.com/5158896/127429804-825f49d7-8930-4a62-8b42-882f05176d52.png)

**Comment by rainprob** [comment]
@fxia22 
I tried to download cuda11.2 for runtime that has the same version as cuda driver version. But the results shown in web_ui is still the same. 
![11](https://user-images.githubusercontent.com/63865505/127635164-f8f37f84-1f08-4403-9aaa-b54e604fd4ef.PNG)

**Comment by fxia22** [comment]
@rainprob did you build the code again? You would need to run `./build.sh build_local` in GibsonEnv folder.

**Comment by rainprob** [comment]
@fxia22 yes, but the buiding result shows some warnings. I am not sure if the warnings mean building failure and will lead a wrong camera output.

**Comment by rainprob** [comment]
@fxia22 
![001](https://user-images.githubusercontent.com/63865505/127977370-28d74fa8-9f6a-48b2-acc9-055886c573d3.PNG)

**Comment by lhy0807** [comment]
same here
